bigquery/src/main/java/org/apache/zeppelin/bigquery/bigQueryInterpreter.java

```diff
@@ -0,0 +1,340 @@
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.json.jackson2.JacksonFactory;
+
+import com.google.api.services.bigquery.Bigquery;
+import com.google.api.services.bigquery.BigqueryScopes;
+import com.google.api.client.json.GenericJson;
+import com.google.api.services.bigquery.Bigquery.Datasets;
+import com.google.api.services.bigquery.BigqueryRequest;
+import com.google.api.services.bigquery.model.DatasetList;
+import com.google.api.services.bigquery.model.Job;
+import com.google.api.services.bigquery.model.TableCell;
+import com.google.api.services.bigquery.model.TableFieldSchema;
+import com.google.api.services.bigquery.model.TableRow;
+import com.google.api.services.bigquery.model.TableSchema;
+import com.google.api.services.bigquery.Bigquery.Jobs.GetQueryResults;
+import com.google.api.services.bigquery.model.GetQueryResultsResponse;
+import com.google.api.services.bigquery.model.QueryRequest;
+import com.google.api.services.bigquery.model.QueryResponse;
+import com.google.api.services.bigquery.model.JobCancelResponse;
+import com.google.gson.Gson;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.util.Properties;
+import java.util.Set;
+
+import org.apache.zeppelin.interpreter.Interpreter;
+import org.apache.zeppelin.interpreter.InterpreterContext;
+import org.apache.zeppelin.interpreter.InterpreterPropertyBuilder;
+import org.apache.zeppelin.interpreter.InterpreterResult;
+import org.apache.zeppelin.interpreter.InterpreterResult.Code;
+import org.apache.zeppelin.interpreter.thrift.InterpreterCompletion;
+import org.apache.zeppelin.scheduler.Scheduler;
+import org.apache.zeppelin.scheduler.SchedulerFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.base.Function;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import com.google.common.collect.Sets.SetView;
+import java.io.PrintStream;
+import java.io.Reader;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+/**
+ * BigQuery interpreter for Zeppelin.
+ * 
+ * <ul>
+ * <li>{@code zeppelin.bigquery.project_id} - Project ID in GCP</li>
+ * <li>{@code zeppelin.bigquery.wait_time} - Query Timeout in ms</li>
+ * <li>{@code zeppelin.bigquery.max_no_of_rows} - Max Result size</li>
+ * </ul>
+ * 
+ * <p>
+ * How to use: <br/>
+ * {@code %bigquery.sql<br/>
+ * {@code
+ *  SELECT departure_airport,count(case when departure_delay>0 then 1 else 0 end) as no_of_delays 
+ *  FROM [bigquery-samples:airline_ontime_data.flights] 
+ *  group by departure_airport 
+ *  order by 2 desc 
+ *  limit 10
+ * }
+ * </p>
+ * 
+ */
+
+
+public class bigQueryInterpreter extends Interpreter {
```

Comment:
Looks like a typo here, but it violates Java [class naming convention](https://google.github.io/styleguide/javaguide.html#s5.2.2-class-names)